src/test/java/com/google/api/codegen/testdata/nodejs_doc_smoke_test_library.baseline

```diff
@@ -0,0 +1,46 @@
+============== file: smoke-test/library_service_smoke_test.js ==============
+/*
+ * Copyright 2017, Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+describe('LibraryServiceSmokeTest', function() {
+  it('successfully makes a call to the service', function(done) {
+    if (!process.env.SMOKE_TEST_PROJECT) {
+      throw new Error("Usage: SMOKE_TEST_PROJECT=<project_id> node #{$0}");
+    }
+    var projectId = process.env.SMOKE_TEST_PROJECT;
+
+    var library = require('../src').v1({
+      // optional auth parameters.
+    });
+    var formattedName = library.bookPath("testShelf-" + Date.now().toString(), projectId);
+    var rating = libraryV1.Book.Rating.GOOD;
+    var book = {
+        rating : rating
+    };
+    var request = {
+        name: formattedName,
+        book: book
+    };
+    library.updateBook(request).then(function(responses) {
+        var response = responses[0];
+        // doThingsWith(response)
```

Comment:
Remove this comment.
This is a smoke test, so we just need to print the response to stdout.